internal/manifests/collector/poddisruptionbudget_test.go

```diff
@@ -29,38 +29,84 @@ import (
 )
 
 func TestPDB(t *testing.T) {
-	type test struct {
-		name           string
-		MinAvailable   *intstr.IntOrString
+	type expected struct {
```

Comment:
i swapped around where the defaulting is happening so now we should test that here.